data_internet.go

```diff
@@ -2,6 +2,8 @@ package gofakeit
 
 // DataInternet consists of various internet information
 var DataInternet = map[string][]string{
-	"browser":       {"firefox", "chrome", "internetExplorer", "opera", "safari"},
-	"domain_suffix": {"com", "biz", "info", "name", "net", "org"},
+	"browser":           {"firefox", "chrome", "internetExplorer", "opera", "safari"},
+	"domain_suffix":     {"com", "biz", "info", "name", "net", "org"},
+	"http_method":       {"HEAD", "GET", "POST", "PUT", "PATCH", "DELETE"},
+	"http_method_lower": {"head", "get", "post", "put", "patch", "delete"},
```

Comment:
Not sure if we need lower. If someone wants the lower version they can just strings.toLower() themselves

Reply:
You're right. it seems a needless. I'll remove it